app/screens/ScheduleScreen/ScheduleScreen.tsx

```diff
@@ -228,10 +322,32 @@ const $container: ViewStyle = {
 }
 
 const $list: ContentStyle = {
+  paddingTop: 48 + spacing.extraLarge + spacing.extraSmall,
```

Comment:
What's the 48 value here? Can we move it to a named constant if it's not part of the spacing scale?

Reply:
I saw that 48 was left as a value in the `paddingBottom` in the one of the styles so I thought we agreed to leave it as is in the previous PRs or something. I can definitely move it as a named constant, for sure 👍🏼